go/obscuronode/host/host.go

```diff
@@ -51,7 +55,7 @@ type Node struct {
 	ID common.Address
 
 	P2p          P2P
-	ethereumNode obscurocommon.L1Node
+	ethereumNode l1client.Client
```

Comment:
better to rename this field to "ethConnector"

Reply:
Done 👍